src/Microsoft.Kusto.ServiceLayer/Microsoft.Kusto.ServiceLayer.csproj

```diff
@@ -35,14 +35,14 @@
 
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.OperationalInsights" />
+    <PackageReference Include="Microsoft.Data.SqlClient" VersionOverride="5.2.0" />
```

Comment:
@cheenamalhotra recommended upgrading to 5.2.0 to resolve MDS issues with .NET 8

Reply:
Are you trying to limit the scope of the update? I think it'd be better to just update everything to the new version instead unless there's some blocking issue that would make that not be possible.

@cheenamalhotra thoughts?

Reply:
I'm fine with updating everything for consistency, especially since net7.0 won't be supported anymore. Some people just had concerns about unnecessary updates in other things like Batch Parser

Reply:
Since it's just 5.1.4 -> 5.2.0 I think that should be fine.

@Matteo-T any concerns on your end about updating sqlclient for ManagedBatchParser?